src/core/stdc/errno.d

```diff
@@ -1757,7 +1757,7 @@ else version (Solaris)
     enum ECONNREFUSED =   146     /** Connection refused */;
     enum EHOSTDOWN =      147     /** Host is down */;
     enum EHOSTUNREACH =   148     /** No route to host */;
-    enum EWOULDBLOCK =    EAGAIN;      /** Resource temporarily unavailable     */;
+    enum EWOULDBLOCK =    EAGAIN  /** Resource temporarily unavailable */;
```

Comment:
That's a weird wait to document symbol though. 👍 for making it consistent, but does it even work ?

Reply:
That stood out to me too, but I think fix in a separate PR.

Reply:
No, those comments need to be after the semicolon but thats for another PR

Reply:
I don't know if those DDocs are used, that's in `version(Solaris)`. Probably it was that way in the original.

Reply:
See #3088